lib/tags/script_tags.rb

```diff
@@ -0,0 +1,12 @@
+module ScriptTags
+  include Radiant::Taggable
+
+  desc 'Econ-Engine javascript tag with the key for cache invalidation'
+  tag 'ee_script_tag' do |tag|
+    version_uri = "#{PageMountCalculator.new(@request).ecom_engine_url}/version.txt"
+    response = Net::HTTP.get_response(URI.parse(version_uri))
```

Comment:
I'm thinking this would be good to have in a '1 minute' cache. That way instead of requesting it 2-3 times per page, which over a minute might be thousands of times, we can just do it once.

A 1 minute cache would at least reduce the number of our problems to 1 minute AND my understanding is we can still have issues anyway during the deployment window as we can have OLD js but then the user submits to new EcomEngine worker (and vice versa).

This seems like a good compromise but ensures no matter how many users on our sites we'll only request ecom engine version once per minute.

Your thoughts?